util/args_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"io/ioutil"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ReadArgsFile", func() {
+
+	It("returns an error when an invalid file name is passed", func() {
+		_, err := ReadArgsFile("/foo/blah")
+		Expect(err).To(HaveOccurred())
+	})
+
+	Context("with valid file", func() {
+		DescribeTable("", func(input string, size int) {
+			f, err := ioutil.TempFile(os.TempDir(), "read_file_args")
+			Expect(err).NotTo(HaveOccurred())
+
+			err = ioutil.WriteFile(f.Name(), []byte(input), 0644)
+			Expect(err).NotTo(HaveOccurred())
+
+			defer f.Close()
+
+			args, _ := ReadArgsFile(f.Name())
+			Expect(args).To(HaveLen(size))
+		},
+			Entry("valid with no spaces", `
+key=value
+foo=bar
+`, 2),
+			Entry("valid with spaces", `
+key = value
+foo= bar
+bloop =blah
+`, 3),
+			Entry("valid with empty values", `
+key =
```

Comment:
Is this valid 
```
key =
```
I would imagine you always have to provide a value.

Reply:
The idea behind this was if the crashd script executor intentionally wants an empty value for a key.